packages/expo-font/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Add additional default for project ID in notification options ([#33359](https://github.com/expo/expo/pull/33359) by [@bradjones1](https://github.com/bradjones1))
```

Comment:
I believe this was added to the incorrect changelog. Instead this needs to be in the `expo-notifications` changelog.